app/assets/src/components/views/amr_heatmap/AMRHeatmapView.jsx

```diff
@@ -124,6 +124,17 @@ export default class AMRHeatmapView extends React.Component {
     return maxValues;
   }
 
+  hasDataToDisplay(maxValues) {
+    let hasData = false;
+    Object.entries(maxValues).forEach(metric => {
```

Comment:
Not sure what is happenings here. I believe Object.entries is used for objects... 
Is maxValues an Object? If so you should use `keys()` or `values()` to be more clear.